pnc-model/src/main/java/org/jboss/pnc/model/BuildConfiguration.java

```diff
@@ -54,9 +56,11 @@
     @ManyToOne(cascade = { CascadeType.MERGE, CascadeType.PERSIST, CascadeType.REFRESH })
     private ProductVersion productVersion;
```

Comment:
We replaced productversionsprojects with buildconfigurationset, but I don't think we specifically said we would remove this field.  However, I do agree that it should be removed, and there is a jira for it: https://projects.engineering.redhat.com/browse/NCL-406